server.go

```diff
@@ -97,7 +102,7 @@ func getConfig() *config {
 
 	//Initialize validator
 	toValidate := map[string]string{}
-	toValidate["aud"] = "api://default"
+	toValidate["aud"] = clientAudience
```

Comment:
`audience`

Reply:
Cool, suggestions made sense, updated them, please review and kind regards, Rogier